api/src/routing.js

```diff
@@ -149,52 +150,11 @@ app.use(
   )
 );
 
-app.use(
-  helmet({
-    // runs with a bunch of defaults: https://github.com/helmetjs/helmet
-    hpkp: false, // explicitly block dangerous header
-    contentSecurityPolicy: {
-      directives: {
-        defaultSrc: [`'none'`],
-        fontSrc: [`'self'`],
-        manifestSrc: [`'self'`],
-        connectSrc: [
-          `'self'`,
-          environment.buildsUrl + '/',
-          'maps.googleapis.com' // used for enketo geopoint widget
-        ],
-        childSrc:  [`'self'`],
-        formAction: [`'self'`],
-        imgSrc: [
-          `'self'`,
-          'data:', // unsafe
-          'blob:',
-          '*.openstreetmap.org', // used for enketo geopoint widget
-        ],
-        mediaSrc: [
-          `'self'`,
-          'blob:',
-        ],
-        scriptSrc: [
-          `'self'`,
-          // Explicitly allow the telemetry script setting startupTimes
-          `'sha256-B5cfIVb4/wnv2ixHP03bHeMXZDszDL610YG5wdDq/Tc='`,
-          // AngularJS and several dependencies require this
-          `'unsafe-eval'`,
-          // Allow Enketo onsubmit form attribute
-          // https://github.com/medic/cht-core/issues/6988
-          `'unsafe-hashes'`,
-          `'sha256-2rvfFrggTCtyF5WOiTri1gDS8Boibj4Njn0e+VCBmDI='`,
-        ],
-        styleSrc: [
-          `'self'`,
-          `'unsafe-inline'` // angular-ui-bootstrap
-        ],
-      },
-      browserSniff: false,
-    },
-  })
-);
+app.use((req, res, next) => {
+  contentSecurityPolicy
+    .get(req, res)
+    .then(policy => helmet(policy)(req, res, next));
```

Comment:
I didn't find a way to mock `helmet`, so using `helmet` here instead of in contentSecurityPolicy file to cover the CSP config with unit tests